recipes/oommf/bld.bat

```diff
@@ -0,0 +1,5 @@
+curl -o oommf.zip  "http://math.nist.gov/oommf/dist/oommf12b0_20160930_86_x64.zip"
```

Comment:
But @isuruf said that the files are 'downloaded and extracted'. So we don't need the unzip command, but instead need to move all unzipped files into `/opt`?